### ISSUE TITLE
Support Gradle 8+

### ISSUE THREAD

**Issue by adolfogc** [LinusU/react-native-get-random-values#50]
Hi, I'm using RN 0.72.6 and Gradle 8, the project fails to build due to namespace issue related to Gradle. The issue can be fixed by adding the following to `build.gradle`:

```diff
     buildToolsVersion safeExtGet('buildToolsVersion', '30.0.3')
 
+    namespace "org.linusu"
+
     defaultConfig {
```

**Comment by adolfogc** [comment]
I now see that a PR was already proposed to fix this: https://github.com/LinusU/react-native-get-random-values/pull/48

**Comment by LinusU** [comment]
Fixed in [🚢 1.10.0 / 2023-11-20](https://github.com/LinusU/react-native-get-random-values/releases/tag/v1.10.0)
